### ISSUE TITLE
Adjust contents of one binding with another

### ISSUE THREAD

**Issue by chrisl212** [vega/vega-lite#6485]
This is based on a [StackOverflow](https://stackoverflow.com/questions/61754243/altair-controlling-one-dropdown-using-another) question I asked today. Here's the question for context:

> I am using Altair to plot a dataset, and I am using selections to filter the data based on two criteria. I'll use the terms 'make' and 'model' (as in cars) to refer to these. These are not the actual names of the data, but the situation is analogous.
> 
> I'd like it so that, when a user selects a 'make' (let's say Cadillac), the 'model' dropdown will adjust its options to different Cadillac models (Escalade, CTS, etc.). 
> 
> As it currently stands, I have two dropdowns successfully filtering the data using the following
> 
>     make_dropdown = alt.binding_select(options=makes)
>     make_select   = alt.selection_single(fields=['make'], bind=make_dropdown, name='Make', init={'make': makes[0]})
> 
>     model_dropdown = alt.binding_select(options=models)
>     model_select   = alt.selection_single(fields=['model_name'], bind=model_dropdown, name='Model', init={'model_name': models[0]})
> However, the `model_dropdown` always shows every model for every make. After searching the Altair documentation, I can't find an easy way to filter its options based on `make_select`. Does anyone know how to do this?

The answer to this question is that, as it stands, this is not possible using Altair/Vega-lite. With that being said, could this feature be added? More specifically, filtering the values of a (in this case) dropdown based on the value selected in another one?

**Comment by domoritz** [comment]
I don’t think this is possible right now but we can consider it for the next version of selections. Cc @arvind
